lib/async.js

```diff
@@ -15,12 +15,21 @@ function findAllConfigs(options) {
         });
 
         return Promise.all(realFiles.map(function(file) {
-            return fsHelpers.read(file).then(function(content) {
-                var parsedConfig = nodeEval('(' + content + ')');
+            var parsedConfig;
 
+            try {
+                parsedConfig = require(file);
                 parsedConfig.__source = file;
-                return parsedConfig;
-            });
+
+                return Promise.resolve(parsedConfig);
+            } catch (err) {
+                return fsHelpers.read(file).then(function(content) {
+                    parsedConfig = nodeEval('(' + content + ')');
```

Comment:
I'd add a check here. Literally there are 3 cases: JSON, JS-object, and JS code with `exports` or `module.exports` variable. The first 2 cases can be evaluated with the existed code. For JS-code you need to pass code to `nodeEval` without modifications (without wrapping to parenthesises).

So I'd made a little fix in this code, removed try-catch and required, etc.

Btw, do you tried to prepend `0,` to code instead of wrapping to parens?